Sources/PackageDescription4/BuildSettings.swift

```diff
@@ -8,7 +8,7 @@
  See http://swift.org/CONTRIBUTORS.txt for Swift project authors
 */
 
-/// The build configuration such as debug or release.
+/// The build configuration for a target.
```

Comment:
"for a target" might not be accurate or future proof so let's just leave this one as is.